nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -92,6 +93,39 @@
         .defaultValue("true")
         .required(true)
         .build();
+    public static final PropertyDescriptor PROXY_HOST = new PropertyDescriptor.Builder()
```

Comment:
I honestly don't have any opinion on this subject but I agree that if we do not add SOCKS option to SFTP processors, then it's OK to have the corresponding properties in SFTPTransfer.